src/modules/to-do-UI.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable consistent-return */
+
+import Task from './AddTask';
+import { tasks, deleteItem } from './removeItem.js';
+import dynamicLI from './dynamicUI';
+
+function listItems(task) {
+  const list = document.querySelector('.list');
+
+  const input = document.createElement('input');
+  input.setAttribute('type', 'checkbox');
+  input.classList.add('checkbox');
+
+  const li = document.createElement('li');
+  li.classList.add('li');
+
+  const line = document.createElement('hr');
+  line.classList.add('line');
+
+  const bin = document.createElement('i');
+  bin.setAttribute('class', 'fa-solid fa-trash');
+  bin.classList.add('icon', 'hidden', 'remove');
+
+  const ellipsis = document.createElement('i');
+  ellipsis.setAttribute('class', 'fa-solid fa-ellipsis-vertical');
+  ellipsis.classList.add('icon2');
+
+  const description = document.createElement('p');
+  description.innerHTML = task.description;
+
+  const inputParaSpan = document.createElement('span');
+  inputParaSpan.classList.add('span');
+  inputParaSpan.append(input, description, ellipsis, bin);
+
+  li.append(inputParaSpan, line);
+
+  list.append(li);
+}
+
+export default listItems;
+
+export function newTask() {
+  const inputField = document.querySelector('.input');
+  inputField.addEventListener('keydown', (event) => {
+    if (event.key === 'Enter') {
+      const inputValue = inputField.value;
+      inputField.value = '';
+      if (inputValue === '') {
+        return false;
+      }
+      const task = new Task(inputValue, false, tasks.length + 1);
+      listItems(task);
+      deleteItem();
+      tasks.push(task);
+      return localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  });
+}
```

Comment:
- [ ] By default, new tasks should have the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5).
- All the added tasks indexes are set to value 1. kindly work on that: 
<img width="350" alt="image" src="https://user-images.githubusercontent.com/77190203/167268404-e354072e-230f-42f3-8c18-584714b50c65.png">